databases/relational-databases.md

```diff
@@ -32,67 +33,22 @@ To understand, query, and insert information into a relational database, a techn
 ### Specific Things To Teach
 
 - The specific problems a database solves
-  - Persistence of information
```

Comment:
This is good info; it just doesn't belong in 'things to teach."  Can move the info into ### Lesson, please?

Reply:
so do I need to remove things to teach if I add it into lessons?

Reply:
No

Reply:
Look at the template to see how each section is used

Reply:
Things to teach is perfect, but the things you removed should now go in lesson.